packages/create-instance/add-slots.js

```diff
@@ -21,23 +23,54 @@ function createElementFromAdvancedString (slotValue, vm) {
   return elem
 }
 
-function createElement (slotValue: string | Object, vm) {
+function validateEnvironment (_window, compileToFunctions): void {
+  if (!compileToFunctions) {
+    throwError('vueTemplateCompiler is undefined, you must pass components explicitly if vue-template-compiler is undefined')
+  }
+  if (typeof _window === 'undefined') {
+    throwError('the slots string option does not support strings in server-test-uitls.')
+  }
+  if (_window.navigator.userAgent.match(/PhantomJS/i)) {
+    throwError('the slots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
+  }
+}
+
+function addSlotToVm (vm: Component, slotName: string, slotValue: Component | string | Array<Component> | Array<string>): void {
```

Comment:
Could you move the slotValue into its own type:

`type SlotValue = Component | string | Array<Component> | Array<string>`

Reply:
Thank you for pointng out.
I added a type.